docker/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Set some vars
+prepopvols=("config" "mods" "art") # these are folders which contain runtime needed files, and need to be represented in the host
+bbspath=/enigma-bbs # install location
+bbsstgp=/enigma-bbs-pre # staging location for prepopvals
+configname=config.hjson # this is the default name, this script is intended for easy get-go - make changes as needed
+
+# Setup happens when there is no existing config file
+if [[ ! -f $bbspath/config/$configname ]]; then
+    for dir in "${prepopvols[@]}"
+    do
+        if [ -n "$(find "$bbspath/$dir" -maxdepth 0 -type d -empty 2>/dev/null)" ]; then
+            cp -rpn $bbsstgp/$dir/* $bbspath/$dir/
+        else
+            echo "WARN skipped $bbspath/$dir - vol Not empty/not a new setup - possible bad state"
```

Comment:
nit: `WARN: Skipped $BBS_ROOT/$DIR: Volume not empty or not a new setup; Possible bad state"

...and maybe some blurb echoed out about actionable steps

Reply:
done in latest commit